Bottom: 4fa1c50e2dd445d88dac3dfe5d57a5bf9550a108
Top:    2fbac692b91c079a462a0e0f3ec257c4ad33d080
Author: Pavel Roskin <proski@gnu.org>
Date:   2013-11-04 10:55:29 -0500

Move $makeflags into $make

---

diff --git a/.farmrc b/.farmrc
index d2b7400..ba873a1 100644
--- a/.farmrc
+++ b/.farmrc
@@ -4,5 +4,5 @@ v2.6.22 v2.6.23 v2.6.24 v2.6.25 v2.6.26 v2.6.27 v2.6.28 v2.6.29 v2.6.30
 v2.6.31 v2.6.32 v2.6.33 v2.6.34 v2.6.35 v2.6.36 v2.6.37 v2.6.38 v2.6.39
 v3.0 v3.1 v3.2 v3.3 v3.4 v3.5 v3.6 v3.7 v3.8 v3.9 v3.10 v3.11 v3.12
 linux-next/master"
-makeflags=-j11
+make="make -j11"
 export EXTRA_CFLAGS=-Werror-implicit-function-declaration
diff --git a/farm-clean b/farm-clean
index 1ab0d27..18e7aab 100755
--- a/farm-clean
+++ b/farm-clean
@@ -7,5 +7,5 @@ for tree in $farm/*; do
 	test -d $tree/Documentation || continue
 	cd $tree
 	echo "Cleaning $tree"
-	make distclean
+	$make distclean
 done
diff --git a/farm-compile b/farm-compile
index 1b8f8aa..f506c6f 100755
--- a/farm-compile
+++ b/farm-compile
@@ -26,9 +26,9 @@ for bintree in $farm/*; do
 	test -n "$arch" || continue
 	log=Log-`basename $bintree`
 	rm -f $log $log-ok $log-error
-	make clean KERNELRELEASE=0 >/dev/null
+	$make clean KERNELRELEASE=0 >/dev/null
 	echo -n "Checking $bintree "
-	if make $makeflags HOSTCC="gcc -DPATH_MAX=4096" CC="gcc-4.4 $ccflags" \
+	if $make HOSTCC="gcc -DPATH_MAX=4096" CC="gcc-4.4 $ccflags" \
 	     ARCH=$arch KBUILD=$bintree KERNELPATH=$bintree C=2 \
 	     CF="-D__CHECK_ENDIAN__" $target >$log 2>&1; then
 		mv $log $log-ok
diff --git a/farm-config32 b/farm-config32
index c1604fc..6f6e4bd 100755
--- a/farm-config32
+++ b/farm-config32
@@ -13,7 +13,7 @@ for srctree in $farm/*; do
 	rm -rf $bintree-bad
 	mkdir $bintree
 	cd $srctree
-	make $makeflags HOSTCC="gcc -DPATH_MAX=4096" KBUILD_OUTPUT=$bintree \
+	$make HOSTCC="gcc -DPATH_MAX=4096" KBUILD_OUTPUT=$bintree \
 		CONFIG_WIRELESS_EXT=y CONFIG_NET_RADIO=y CONFIG_IPW2100=y \
 		CONFIG_CRYPTO=y ARCH=$arch defconfig
 	cd $bintree
@@ -21,8 +21,8 @@ for srctree in $farm/*; do
 	sed -i -e 's/# CONFIG_WIRELESS_EXT is not set/CONFIG_WIRELESS_EXT=y/' .config
 	sed -i -e 's/# CONFIG_IPW2100 is not set/CONFIG_IPW2100=y/' .config
 	sed -i -e 's/# CONFIG_CRYPTO is not set/CONFIG_CRYPTO=y/' .config
-	yes "" | make $makeflags HOSTCC="gcc -DPATH_MAX=4096" ARCH=$arch oldconfig
-	if make $makeflags CC="gcc-4.4 $ccflags" HOSTCC="gcc -DPATH_MAX=4096" \
+	yes "" | $make HOSTCC="gcc -DPATH_MAX=4096" ARCH=$arch oldconfig
+	if $make CC="gcc-4.4 $ccflags" HOSTCC="gcc -DPATH_MAX=4096" \
 	     ARCH=$arch modules_prepare >Log-prepare 2>&1; then
 		date > $bintree/configured-$arch
 	else
diff --git a/farm-config64 b/farm-config64
index b4dc14b..d30dd44 100755
--- a/farm-config64
+++ b/farm-config64
@@ -13,7 +13,7 @@ for srctree in $farm/*; do
 	rm -rf $bintree-bad
 	mkdir $bintree
 	cd $srctree
-	make $makeflags HOSTCC="gcc -DPATH_MAX=4096" KBUILD_OUTPUT=$bintree \
+	$make HOSTCC="gcc -DPATH_MAX=4096" KBUILD_OUTPUT=$bintree \
 		CONFIG_WIRELESS_EXT=y CONFIG_NET_RADIO=y CONFIG_IPW2100=y \
 		CONFIG_CRYPTO=y ARCH=$arch defconfig
 	cd $bintree
@@ -21,8 +21,8 @@ for srctree in $farm/*; do
 	sed -i -e 's/# CONFIG_WIRELESS_EXT is not set/CONFIG_WIRELESS_EXT=y/' .config
 	sed -i -e 's/# CONFIG_IPW2100 is not set/CONFIG_IPW2100=y/' .config
 	sed -i -e 's/# CONFIG_CRYPTO is not set/CONFIG_CRYPTO=y/' .config
-	yes "" | make $makeflags HOSTCC="gcc -DPATH_MAX=4096" ARCH=$arch oldconfig
-	if make $makeflags CC="gcc-4.4 $ccflags" HOSTCC="gcc -DPATH_MAX=4096" \
+	yes "" | $make HOSTCC="gcc -DPATH_MAX=4096" ARCH=$arch oldconfig
+	if $make CC="gcc-4.4 $ccflags" HOSTCC="gcc -DPATH_MAX=4096" \
 	     ARCH=$arch modules_prepare >Log-prepare 2>&1; then
 		date > $bintree/configured-$arch
 	else
